changelog.yaml

```
@@ -21,6 +21,12 @@
 - version: "NEXT"
   date: TBD
   changes:
+  - title: Upgrade github.com/aws/aws-sdk-go
```

For dep updates we don't need to be specific in the title:
```suggestion
  - title: Upgrade Go dependencies
```